braintree/braintree-ios-drop-in
# Update UI Tests

### Summary of changes

 - Cleanup `XCTestCase` extensions
     - We were not using `file` or `line` so can simplify these methods quite a bit
     - Updated these methods to match the implementation used in `braintree_ios`
 - Bump timeouts to 20s
 - Remove 3DS v1 UITests
     - 3DSv1 is no longer supported so understandably 3DS v1 UITests were failing
     - 3DSv2 UITests exist in the `BraintreeDropIn_ThreeDSecure_2_UITests` class
 - Update missing postal code field on test

 ### Checklist


### Authors
> List GitHub usernames for everyone who contributed to this pull request.

- @jaxdesmarais
